docker/Dockerfile.webpack

```diff
@@ -1,24 +1,20 @@
 ARG NODE_VERSION=10.15-alpine
-FROM node:$NODE_VERSION AS webpack-base
+FROM node:$NODE_VERSION
 
-ARG GIT_COMMIT_SHA
+ARG NODE_VERSION
 
 LABEL org.label-schema.vcs-url="https://github.com/metabrainz/listenbrainz-server.git" \
-      org.label-schema.vcs-ref=$GIT_COMMIT_SHA \
+      org.label-schema.vcs-ref="" \
```

Comment:
no labels in the node image, so maybe we don't need this one either
alternatively - we keep it in case they add it at a later stage?